DuckDuckGo/PrivacyProtectionFooterController.swift

```diff
@@ -22,28 +22,42 @@ import UIKit
 import Core
 
 class PrivacyProtectionFooterController: UIViewController {
-
+    
+    private struct DisplayConstants {
+        static let tallScreenHeight: CGFloat = 746
+        static let tallLeaderboardTopPadding: CGFloat = 108
+        static let defaultLeaderboardTopPadding: CGFloat = 8
+    }
+    
     @IBOutlet weak var privacyProtectionView: UIView!
     @IBOutlet weak var privacyProtectionSwitch: UISwitch!
     @IBOutlet weak var leaderboard: TrackerNetworkLeaderboardView!
+    @IBOutlet weak var leaderboardTopConstraint: NSLayoutConstraint!
 
-    let configuration: ContentBlockerConfigurationStore = AppDependencyProvider.shared.storageCache.current.configuration
-
+    fileprivate var domain: String?
+    fileprivate var contentBlockerConfiguration: ContentBlockerConfigurationStore!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         leaderboard.didLoad()
+        update(forHeight: view?.frame.size.height ?? 0)
     }
-
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        update()
+    
+    func update(forHeight height: CGFloat) {
```

Comment:
I'm not clear why this change was made?  The screen now seems to be badly laid out. 

![Simulator Screen Shot - iPhone 11 Pro - 2019-10-28 at 11 02 44](https://user-images.githubusercontent.com/81197/67674116-9487e500-f973-11e9-832e-2f5d22b5b3ad.png)

![Simulator Screen Shot - iPhone 11 Pro - 2019-10-28 at 11 01 32](https://user-images.githubusercontent.com/81197/67674117-95207b80-f973-11e9-9e75-8a8385db3f51.png)